internal/cloud/gcp/compute.go

```diff
@@ -60,8 +62,33 @@ func (g *GCP) ComputeImageImport(ctx context.Context, bucket, object, imageName,
 		"-timeout=7000s",
 		"-client_id=api",
 	}
+
 	if region != "" {
 		buildStepArgs = append(buildStepArgs, fmt.Sprintf("-storage_location=%s", region))
+
+		// Set the region to be used by the daisy workflow when creating resources
+		// If not specified, the workflow seems to always default to
+
+		// The Region passed as the argument is a Google Storage Region, which can be a multi or dual region.
+		// Multi and Dual regions don't work with GCE API, therefore we need to get the list of GCE regions
+		// that they map to. If the passed Region is not a multi or dual Region, then the returned slice contains
+		// only the Region passed as an argument.
+		gceRegions, err := g.storageRegionToComputeRegions(ctx, region)
+		if err != nil {
+			return nil, fmt.Errorf("failed to translate Google Storage Region to GCE Region: %v", err)
+		}
+
+		// Pick a random GCE Region to be used by the image import workflow
+		rand.Seed(time.Now().UnixNano())
```

Comment:
I always prefer `crypto/rand` as a seed because it doesn't have a global state which is always weird. On the other hand, it's overkill for choosing a region. What do you think?

Reply:
I think that it makes sense to be consistent across the whole composer code base. Thanks.

Reply:
This is changed to `crypto/rand`.